project-base/.github/workflows/run-checks-tests.yaml

```diff
@@ -35,3 +35,48 @@ jobs:
                 with:
                     name: acceptance-logs
                     path: ./var/log
+    build-whole-application-for-single-domain:
+        name: Build application and run standards checks and tests on single domain
+        runs-on: 'ubuntu-20.04'
+        steps:
+            -   name: GIT checkout branch - ${{ github.ref }}
+                uses: actions/checkout@v4
+                with:
+                    ref: ${{ github.ref }}
+            -   name: Configure application
+                run: echo 1 | ./scripts/configure.sh
+            -   name: Remove other domains than first one from configs
+                run: |
+                    # delete all domain URLs except URLs for domain with ID 1 in app/config/domains.yaml
+                    sed -i '/id: 2/{:a;Q}' app/config/domains_urls.yaml
+                    sed -i '/id: 2/{:a;Q}' app/config/domains.yaml
+
+                    # sed is removing all content from match to end of file, unfortunately it adds new line (\n) after the replacement
+                    # more info: https://stackoverflow.com/questions/50559170/delete-last-line-break-using-sed
+                    # when there stays more than one last line the build will fail because of our coding standards
+                    truncate -s -1 app/config/domains_urls.yaml
+                    truncate -s -1 app/config/domains.yaml
+            -   name: Install composer dependencies
+                run: docker-compose exec -T php-fpm composer install
+            -   name: Ensure Symfony 4 version
```

Comment:
this is probably not necessary anymore - we use Symfony 5 already, right? Or maybe, it is necessary and the name should be changed to `Ensure Symfony 5 version`? :smile: see https://github.com/shopsys/shopsys/commit/e4dd48defd9f32a1fbd24e54d9812061f383afc1

Reply:
I have removed it